src/Form/Row/Row.js

```diff
@@ -20,12 +20,32 @@ const StyledRow = styled.div`
       align-items: center;
     `}
 
+  ${({ direction }) =>
+    direction === "column" &&
+    css`
+      flex-direction: column;
+      align-items: start;
+    `}
+
   ${(theme) => SPACER(theme)}
   ${(theme) => DISPLAY(theme)}
 `;
 
 const StyledLabel = styled(FormLabel)`
   flex: 0 0 30%;
+
+  ${({ direction }) =>
+    direction === "column" &&
+    css`
+      margin-bottom: ${({ theme }) => theme.spacing(3)};
+    `}
+
+  ${({ labelColor }) =>
+    labelColor === "dark" &&
+    css`
+      color: #08042a;
```

Comment:
please use syntax with `theme colors`, current implementation can cause some bugs during colors changes. In our project colors are customizable, so that we should follow the pattern.

In a file `colors.js` you can find your color.
```js

export const KIT_COLORS = {
  primary: {
    main: "#371FEB",
    light: "#DEDAFC",
    accent: "FFFB4D",
  },
  gray: {
    black: "#08042A",
    dark: "#4A485B",
    regular: "#DAD9DD",
    medium: "#9796A2",
    light: "#F7F7F8",
    semiLight: "#F1F1F4",
    white: "#FFFFFF",
  },
  alert: {
    darkRed: "#F15937",
    lightRed: "#FCDFD9",
    darkGreen: "#0EC811",
    lightGreen: "#D9FCD9",
    darkYellow: "#5C5800",
    lightYellow: "#FFFC8A",
  },
};

```

It looks like it is `  gray: {
    black: "#08042A",` - `KIT_COLORS.gray.black`

in `theme.js` file is given:

```js
 theme.palette = {
    ...theme.palette,
    primary: {
      main: colors?.primaryMain || KIT_COLORS.primary.main,
      light: colors?.primaryLight || KIT_COLORS.primary.light,
    },
    secondary: {
      main: colors?.elementsThirdColor || KIT_COLORS.gray.dark,
      light: colors?.elementsSecondaryColor || KIT_COLORS.gray.semiLight,
      dark: colors?.basicFontsColor || KIT_COLORS.gray.black,
    },
    text: {
      primary: colors?.basicFontsColor || KIT_COLORS.gray.black,
      secondary: colors?.secondaryFontColor || KIT_COLORS.gray.medium,
    },

    gray: {
      black: colors?.basicFontsColor || KIT_COLORS.gray.black,
      dark: colors?.elementsThirdColor || KIT_COLORS.gray.dark,
      regular: colors?.secondaryComponentsColor || KIT_COLORS.gray.regular,
      medium: colors?.secondaryFontColor || KIT_COLORS.gray.medium,
      light: colors?.elementsPrimaryColor || KIT_COLORS.gray.light,
      semiLight: colors?.elementsSecondaryColor || KIT_COLORS.gray.semiLight,
      white: KIT_COLORS.gray.white,
    },
    success: {
      main: KIT_COLORS.alert.darkGreen,
      light: KIT_COLORS.alert.lightGreen,
    },
    warning: {
      main: KIT_COLORS.alert.darkYellow,
      light: KIT_COLORS.alert.lightYellow,
    },
    error: {
      main: KIT_COLORS.alert.darkRed,
      light: KIT_COLORS.alert.lightRed,
    },
    common: {
      white: colors?.componentsColor || KIT_COLORS.gray.white,
      black: KIT_COLORS.gray.black,
    },
    background: {
      default: colors?.bgColor || KIT_COLORS.gray.light,
    },
  };

```

so we are seeking `KIT_COLORS.gray.black` and it is under `theme.palette.gray.black`,
so the css line for this is:

`color: ${({ theme }) => theme.palette.gray.black};`

the same is with margins, paddings, etc.

Reply:
ofc :)